### PR TITLE
created - password manager popup aka DiscordDialog.kt

### PR DESCRIPTION
**Pull request by yugeshjain**

### CODE REVIEW

**Comment by Anmol92verma** [comment]
@yugeshjain need screenshots, please resolve the conflicts

**Comment by yugeshjain** [comment]
password_manager_dialog_dark
password_manager_dialog_light

File: app/src/main/java/dev/baseio/discordjetpackcompose/ui/routes/onboarding/commonui/DiscordDialog.kt [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        Dialog(
+            onDismissRequest = { onDismissRequest() },
+            properties = properties
+        ) {
+            Surface(
+                modifier = modifier,
+                shape = shape,
+                color = backgroundColor
+            ) {
+                Column(
```
Comment (Anmol92verma) [inline app/src/main/java/dev/baseio/discordjetpackcompose/ui/routes/onboarding/commonui/DiscordDialog.kt:62]: @yugeshjain This UI Tree can be extracted as Composable methods like Title() Subtitle() DialogOptions() DialogPositive() DialogNegative()

Comment (yugeshjain) [reply app/src/main/java/dev/baseio/discordjetpackcompose/ui/routes/onboarding/commonui/DiscordDialog.kt:62]: Done. Resolved

**Pull request merged**